lib/deppack/modules.js

```diff
@@ -119,103 +188,35 @@ const unexpandedFilePath = filePath => {
-    }
-  }).then(main => {
-    return unexpandedFilePath(main);
-  });
-};
 
+// Module wrapping
 
-const isMain = filePath => getMainCached(filePath) === sysPath.resolve(expandedFilePath(filePath));
-
-const getNewHeader = (moduleName, source, filePath, origPath) => {
+const getHeader = (moduleName, filePath, source) => {
   const brMap = globalBrowserMappings(filePath);
 
   const glob = shims.findGlobals(source);
 
-  if (shims.shouldOverrideModuleName(moduleName)) moduleName = shims.overrideModuleName(moduleName);
-
   if (filePath.indexOf('.json') === -1) {
-    const fbModuleName = generateFileBasedModuleName(filePath);
-    const fbAlias = mediator.npm.noFileBased ? shims.shouldIncludeFileBasedAlias(moduleName) : isMain(filePath) ?
-      aliasDef(fbModuleName, moduleName) + '\n' :
-      '';
-    const fbModuleNameUnexp = generateFileBasedModuleName(origPath);
-    const fbAliasUnexp = mediator.npm.noFileBased ?
-      '' :
-      fbModuleNameUnexp !== fbModuleName ?
-        aliasDef(fbModuleNameUnexp, moduleName) :
-        '';
-
-    const p = origPath.replace(getModuleRootPath(origPath), '').replace('.json', '').replace('.js', '').split(sysPath.sep).slice(0, -1);
-    const p2 = [moduleName].concat(p).join('/');
-    const itemPath = isMain(filePath) ? `, '${p2}/'` : '';
-
     const mappings = JSON.stringify(brMap);
     const name = getModuleFullRootName(filePath);
-    const aliases = (fbAlias + fbAliasUnexp).trim();
-    const aliasesFull = aliases ? '\n\n' + aliases : '';
     return (
 `\nrequire.register("${moduleName}", function(exports, require, module) {
-  require = __makeRelativeRequire(require, ${mappings}, "${name}"${itemPath});
+  require = __makeRelativeRequire(require, ${mappings}, "${name}");
   ${glob}(function() {
     ${source.trim()}
   })();
-});${aliasesFull}`);
+});`);
   } else {
     return definition(moduleName, source);
   }
 };
 
 const generateModule = (filePath, source) => {
-  const expandedPath = expandedFilePath(filePath);
-  const mn = generateModuleName(expandedPath);
-  return getNewHeader(mn, source, expandedPath, filePath);
-};
-
-const slashes = string => string.replace(/\\/g, '/');
-
-const generateModuleName = filePath => {
-  const rp = getModuleRootPath(filePath);
-  const mn = getModuleFullRootName(filePath) +
-    (isMain(filePath) ? '' : filePath.replace(rp, '').replace('.json', '').replace('.js', ''));
-
-  return slashes(mn);
-};
-
-const generateFileBasedModuleName = filePath => {
-  return slashes(getModuleFullRootName(filePath) + filePath.replace(getModuleRootPath(filePath), '').replace('.json', '').replace('.js', ''));
+  return getHeader(generateFileBasedModuleName(filePath), filePath, source);
 };
 
 const makeRequire = (
-`\nvar __makeRelativeRequire = function(require, mappings, pref, fullPath) {
-  if (fullPath) {
-    var req = require;
-    require = function(path) {
-      return req(path.replace('./', fullPath));
-    };
-  }
```

Comment:
This is also no longer needed because if module names are always file-system based, relation expansion will be handled properly by commonjs require itself